homeassistant/strings.json

```diff
@@ -66,6 +66,7 @@
         "already_configured_service": "Service is already configured",
         "already_in_progress": "Configuration flow is already in progress",
         "no_devices_found": "No devices found on the network",
+        "not_internet_accessible": "Your Home Assistant instance needs to be accessible from the internet to receive webhook messages.",
```

Comment:
Should we prefix the key with `webhook_` since we mention webhook in the sentence?

Reply:
Done